src/main/java/games/strategy/triplea/delegate/MustFightBattle.java

```diff
@@ -321,8 +321,8 @@ public void fight(final IDelegateBridge bridge) {
     if (m_stack.isExecuting()) {
       final ITripleADisplay display = getDisplay(bridge);
       display.showBattle(m_battleID, m_battleSite, getBattleTitle(),
-          removeNonCombatants(m_attackingUnits, true, m_attacker, false, false, false),
```

Comment:
This file contains the largest number of changes by far and might be a candidate for extraction to a separate PR.